tests/blackbox_test.go

```diff
@@ -359,6 +321,51 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 	return nil
 }
 
+// Replace all UUID variables (format $uuid<num>) in configs and partitions with an UUID
+func replaceAllUUIDVars(t *testing.T, test types.Test, UUIDmap map[string]string) (types.Test, error) {
+	var err error
+
+	test.Config, err = replaceUUIDVars(t, test.Config, UUIDmap)
+	if err != nil {
+		return test, err
+	}
+
+	for _, disk := range test.In {
```

Comment:
Further refactoring suggestion: break the inner loop into a function and just call that since its the same for the In and Out disks.

Reply:
Good call. Fixed